client/Cargo.toml

```diff
@@ -13,6 +13,7 @@ grpcio = "0.4.3"
 hex = "0.3.2"
 hyper = "0.12.33"
 itertools = "0.8.0"
+mktemp = "0.3"
```

Comment:
any reason why 0.3 vs 0.4? (I know nothing about mktemp crate, just curious :))

Reply:
Blasted the crate